ga/20.0.0.8/oidcProvider/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM websphere-liberty:kernel
+FROM websphere-liberty:20.0.0.8-kernel
```

Comment:
Based on the tags pushed to Dockerhub and what this was before I think it should stay as `websphere-liberty:kernel` for now, though perhaps in the future this should become `websphere-liberty:20.0.0.8-kernel-java8-ibmjava`.